lxd/containers.go

```diff
@@ -25,6 +28,26 @@ import (
 	"gopkg.in/lxc/go-lxc.v2"
 )
 
+// #cgo LDFLAGS: -lutil
+/*
+#include <unistd.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <grp.h>
+#include <pty.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+#include <stdio.h>
+#include <termios.h>
+
+void own_pty(int fd) {
+	if (ioctl(fd, TIOCSCTTY, (char *)NULL) == -1)
+		printf("Failed TIOCSCTTY: %s\n", strerror(errno));
+}
+*/
+import "C"
+
```

Comment:
The main reason for that is that as soon as you have C code, the line numbers returned in error messages become useless as they're relative to the start of the Go code. So I'd prefer we concentrate the Cgo stuff to only one/a couple of files.